### DISCUSSION TITLE
[Challenge] Error with MSAL-Config

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#17254] [Q&A] [answered]
I just tried to beat the speed-challenge from @MartinZikmund but ran into a problem: I could not immediately start the app as the following error happens:
![image](https://github.com/unoplatform/uno/assets/1833242/af1d9edd-ce3d-4520-9744-ce7b8f32e6d9)

(Downgrade of the package Microsoft.Identity-Client from 4.61.3 to 4.60.3 detected)

**Comment by jeromelaban** [comment]
Thanks for the report. @TopperDEL How did you create the app?

**Reply by TopperDEL** [reply]
I'll upload that specific one in a minute - need to cut the video at the end as I forgot to stop recording. :)

**Reply by TopperDEL** [reply]
https://github.com/unoplatform/uno/assets/1833242/2eb55671-c8d3-436f-89ab-cb5b6f63edbd

This my previous attempt that leads to the "downgrade"-issue.

**Comment by TopperDEL** [comment]
I just ran the challenge again and left the Authentication to "None" (I did not choose MSAL). I still got an error during build, but nothing shows up. It just says "1 error":

![image](https://github.com/unoplatform/uno/assets/1833242/25273875-e72e-45ff-9fb9-40e8f22d713f)

This is the video of me trying to beat Martin:

https://github.com/unoplatform/uno/assets/1833242/b859fd9e-d448-415b-afa1-81da6753d0d2

I assume that it got confused with switching the targets. I have chosen "unpackaged" but after my click on run it changed to "packaged" and failed. Maybe the build got cancelled during the switch.

**Reply by TopperDEL** [reply]
I'm still wondering why my machine is so slow:
![image](https://github.com/unoplatform/uno/assets/1833242/9a1bc53e-c203-4a92-a5e9-d4841c37fea9)

Windows runs on a NVMe SSD, the Dev-Drive is on another SSD. Should be much faster.

**Reply by jeromelaban** [reply, marked as answer]
A nuget package that contained a conflicting update was published, it's now been reverted. You should be able to create an app without issues, now.

**Reply by jeromelaban** [reply]
As for the build error, it happens because of VS issues where the projects need to be reloaded when changing target frameworks. You need to wait a few seconds before pressing the debug button.

**Reply by TopperDEL** [reply]
Waiting a few seconds was not an option here...you see....? 🤣

But yeah: thanks for investigating and fixing!
